<launch>
	
	<!-- Launch file for Jackal navigation demo -->
	<include file="$(find jackal_navigation)/launch/odom_navigation_demo.launch" />

	<!-- Launch file for rosbridge server -->
	<include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch" />

	<!-- Run the jackal_rot_server -->
	<node="get_jackal_rot_server" pkg="simple_navigation_goals" type="jackal_rot_server.py" />

	<!-- Run the jackal_pos_server -->
	<node name="jackal_pos_server_node" pkg="simple_navigation_goals" type="jackal_pos_server.py" />

	<!-- Run the jackal_rot_server -->
	<node name="jackal_rot_server_node" pkg="simple_navigation_goals" type="jackal_rot_server.py" />

</launch>
